calendar_test.go

```diff
@@ -360,3 +363,13 @@ func TestIssue52(t *testing.T) {
 		t.Fatalf("cannot read test directory: %v", err)
 	}
 }
+
+func FuzzParseCalendar(f *testing.F) {
+	ics, err := os.ReadFile("testdata/timeparsing.ics")
+	require.NoError(f, err)
+	f.Add(ics)
+	f.Fuzz(func(t *testing.T, ics []byte) {
+		_, err := ParseCalendar(bytes.NewReader(ics))
+		t.Log(err)
```

Comment:
Shouldn't this be t.Errorf?

Reply:
Acknowledged. I haven't used the fuzzing feature myself. Not nearly enough go coding these days.